### DISCUSSION TITLE
Incorrect translation

### DISCUSSION THREAD

**Discussion by davidbartoncz** [woocommerce/woocommerce#53220] [Checkout Flow]
Hi, at the checkout page, the Orders totals, the name "Delivery" stays the same in Czech. So everything is translated, except "Delivery" (then price) and "Delivers to" (with address next to it). It is frustrating, because this cannot be edited, unlike other labels in checkout. I also tried to manually translate this with Loco Translate plugin, with no avail.

**Comment by nielslange** [comment]
Hello @davidbartoncz 👋

Thanks for reaching out. I looked up this issue and noticed that the [string "Delivery" hasn't been translated](https://translate.wordpress.org/projects/wp-plugins/woocommerce/dev/cs/default/?filters%5Bterm%5D=Delivery&filters%5Bterm_scope%5D=scope_any&filters%5Bstatus%5D=current_or_waiting_or_fuzzy_or_untranslated&filters%5Buser_login%5D=&filter=Apply+Filters&sort%5Bby%5D=priority&sort%5Bhow%5D=desc) into Czech yet. Since translations are managed by community members like yourself, you can add the missing translation there. For more details on how to submit translations, please refer to the [Translation Handbook](https://make.wordpress.org/polyglots/handbook/translating/glotpress-translate-wordpress-org).

As for Locco Translate, it appears that this plugin is having issues picking up JavaScript-rendered strings. You could use the plugin [Say what?](https://wordpress.org/plugins/say-what/) instead. Once you've installed the plugin, you can add the desired translation via `/wp-admin/tools.php?page=say_what_admin`. Simply provide the original string, in this case `Delivery`, the text domain, which is `woocommerce` and the desired translation, e.g. `Dodání`. Here's a screenshot of my setting:

<img width="1112" alt="Screenshot 2024-12-03 at 10 48 09" src="https://github.com/user-attachments/assets/ff42de60-1304-4d8e-8254-15e022f75096">

Once you've provided the desired translation, you should see it on your site:

<img width="1153" alt="Screenshot 2024-12-03 at 10 48 57" src="https://github.com/user-attachments/assets/c20280e2-0c37-476a-ae09-d6a63ccb425e">

I'm closing this thread as there isn’t anything actionable for us on this issue. That said, if you have any other questions or suggestions, feel free to create a new issue or thread.
